src/Resources/translations/SonataMediaBundle.de.xliff

```diff
@@ -1,4 +1,4 @@
-<?xml version="1.0" encoding="UTF-8"?>
+<?xml version='1.0' encoding='UTF-8'?>
 <xliff xmlns="urn:oasis:names:tc:xliff:document:1.2" version="1.2">
   <file source-language="en" target-language="de" datatype="plaintext" original="file.ext">
```

Comment:
I think this `origina="file.ext"` does not work properly, but I don't know for sure how to fix it.

Reply:
Because of the removal ?

Reply:
Indeed https://hosted.weblate.org/browse/sonata-project/SonataAdminBundle/fr/

Maybe `original=""` then ?